.github/workflows/buildtest.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Build
         run: dotnet build --configuration Release -v detailed
       - name: Test
-        run: dotnet test /p:CollectCoverage=true /p:ExcludeByFile=\"**/KubernetesClient/generated/**/*.cs\" /p:CoverletOutputFormat="cobertura"
+        run: dotnet test /collect:"Code Coverage;Format=Cobertura"
```

Comment:
@tg123 if would you like to upload the results to codecov, then it's as simple as using https://github.com/codecov/codecov-action and simply passing `files: '*.coverage.xml'`.